spec/capitalize_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../capitalize_decorator'
+require_relative '../decorator'
+
+describe CapitalizeDecorator do
```

Comment:
- [ ] Dear Team, Great job testing almost all your classes, but I can see you have not tested the methods inside the `App` class. this includes methods like `save_data, load_data, create_a_person`. Please remember we need to have unit tests for all the methods in all the classes.
- [ ] Also, Please add a `spec` for the `List` class to test methods which are used to `list_all_books, list_all_people, list_all_rentals`. This methods should be tested in the `list_spec.rb` file.

Reply:
Add unit tests for all the methods in all the classes:
Book
Person
Student
Teacher
Classroom
Rental
all classes that you created while implementing the Decorator design pattern

I don't think there was an instruction to test for the App class. Kindly go through the project requirements to confirm this again,

Reply:
Hi @TadesseDev ,

According to the requirements, we must add unit tests for the following classes: Book, Person, Student, Teacher, Classroom, and Rental. We also have to test all classes that we created while implementing the Decorator design pattern which are TrimmerDecorator and CapitalDecorator. 

Kindly reconsider your decision because we think that we did everything according to the requirements.
Thank you

Reply:
-  Dear @Bhekisile, I noticed the project requirement is a bit ambiguous on my side, I noticed the student project requirement explicitly mentioned which classes to test. on my side, it just mentions to see if `for all the methods in all the classes` has a test. 
![Screen Shot 2023-10-11 at 8 56 27 PM](https://github.com/Bhekisile/School-Library/assets/69077061/d2941ba5-bf47-4849-bada-9b28dac54109)

- I'll mark the required choices as optional, The next reviewer will approve. Please resubmit for a second review.

Reply:
Hi @TadesseDev,

We appreciate your feedback and thank you for understanding.
Thank you